cpp/src/groupby/hash/compute_mapping_indices.cuh

```diff
@@ -159,14 +158,12 @@ CUDF_KERNEL void mapping_indices_kernel(cudf::size_type num_input_rows,
 }
 
 template <class SetRef>
-cudf::size_type max_occupancy_grid_size(cudf::size_type n)
+size_type max_active_blocks_mapping_kernel()
 {
-  cudf::size_type max_active_blocks{-1};
+  size_type max_active_blocks{-1};
```

Comment:
Will libcudf's `size_type` always match what `cudaOccupancyMaxActiveBlocksPerMultiprocessor`?
Seems like want to use the correct type here and not tie it to the size of libcudf columns.
Perhaps this is outside the scope of this PR.